src/pages/home/report/ReportActionCompose/SendButton.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {View} from 'react-native';
+import {runOnJS} from 'react-native-reanimated';
+import {Gesture, GestureDetector} from 'react-native-gesture-handler';
+import PropTypes from 'prop-types';
+import styles from '../../../../styles/styles';
+import themeColors from '../../../../styles/themes/default';
+import Icon from '../../../../components/Icon';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import CONST from '../../../../CONST';
+import Tooltip from '../../../../components/Tooltip';
+import PressableWithFeedback from '../../../../components/Pressable/PressableWithFeedback';
+import updatePropsPaperWorklet from '../../../../libs/updatePropsPaperWorklet';
+import useLocalize from '../../../../hooks/useLocalize';
+
+const propTypes = {
+    /** Whether the button is disabled */
+    isDisabled: PropTypes.bool.isRequired,
+
+    /** Reference to the animated view */
+    animatedRef: PropTypes.func.isRequired,
+
+    /** Sets the isCommentEmpty flag to true */
+    setIsCommentEmpty: PropTypes.func.isRequired,
+
+    /** Submits the form */
+    submitForm: PropTypes.func.isRequired,
+};
+
+function SendButton({isDisabled: isDisabledProp, animatedRef, setIsCommentEmpty, submitForm}) {
+    const {translate} = useLocalize();
+
+    const Tap = Gesture.Tap()
+        .enabled()
```

Comment:
Enabled condition was not preserved `!(isCommentEmpty || isBlockedFromConcierge || disabled || hasExceededMaxCommentLength)` and caused regression https://github.com/Expensify/App/issues/27779